new_project/readme.md

```diff
@@ -0,0 +1,88 @@
+# Github Release Process
```

Comment:
What sort of review process are PRs going to be subjected to? Is it just the owner of the repo, or is it going to have to go much further up the chain?

Reply:
Did this question get answered by the issue checklist?

Reply:
Not exactly. That's for a new repository is it not? I'm thinking about a pull request on an existing repository.

Reply:
Oh, right. I don't know yet, I haven't proposed anything. This PR just proposes a process for releasing new open-source projects. We probably need to address PRs in a different document.

Reply:
There hasn't been much discussion as of yet regarding this topic, but in my opinion anything that is a "release" of some software/code is going to have a larger stakeholder approval process. Pulling code in, such as these PR's, I would think would be handled at the project team level.